src/inference/src/dev/core_impl.hpp

```diff
@@ -251,11 +249,35 @@ class CoreImpl : public InferenceEngine::ICore, public std::enable_shared_from_t
             if (deviceName.find('.') != std::string::npos) {
                 IE_THROW() << "Device name must not contain dot '.' symbol";
             }
-            const auto& value = plugin.second;
-            ov::AnyMap config = any_copy(value.m_default_config);
-            PluginDescriptor desc{value.m_create_plugin_func, config, value.m_create_extension_func};
-            pluginRegistry[deviceName] = desc;
-            add_mutex(deviceName);
+            if (pluginRegistry.find(deviceName) == pluginRegistry.end() {
```

Comment:
I think it should fix the build :)
```suggestion
            if (pluginRegistry.find(deviceName) == pluginRegistry.end()) {
```

Reply:
Thanks, presumably missed that brace while integrating the changes into a clean branch for the PR 👍